Cargo.toml

```diff
@@ -448,34 +448,34 @@ x25519-dalek = "1.2.0"
-move-compiler ={ git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-core-types = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14", features = ["address32"] }
-move-docgen = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-ir-compiler = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-model = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-package = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-prover = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-prover-boogie-backend = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-stackless-bytecode = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-prover-test-utils = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-resource-viewer = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-stackless-bytecode-interpreter = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-stdlib = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-symbol-pool = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-table-extension = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-transactional-test-runner = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-move-unit-test = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14", features = ["table-extension"] }
-move-vm-runtime = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14", features = ["lazy_natives"] }
-move-vm-test-utils = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14", features = ["table-extension"] }
-move-vm-types = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-read-write-set = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
-read-write-set-dynamic = { git = "https://github.com/move-language/move", rev = "318fe9b3314d88aabc3fe641033dad06aaee7c14" }
+move-abigen = { git = "https://github.com/move-language/move", rev = "375359548197a41c58fe2c07e21f9cdd76c7b153" }
```

Comment:
Why are these versions different from what was done in https://github.com/aptos-labs/aptos-core/pull/6134/files?

Reply:
They are different because here we are targeting the [aptos-v1.2.0](https://github.com/move-language/move/compare/aptos-v1.2.0) branch in move repo rather than the aptos-main branch for main.